packages/accordion/tests/react/index.js

```diff
@@ -23,7 +23,7 @@ class AccordionWrapper extends React.Component {
 	render() {
 		return (
 			<Fragment>
-				<AUaccordion header="State controlled accordion open" state={ this.state }>
+				<AUaccordion header="State controlled accordion open" state={ this.state } onOpen={() => {this.changeAccordion('closed')}} onClose={()=> {this.changeAccordion('closed')}}>
```

Comment:
I think adding some spacing between the functions and parameters would spice this up a bit. ( e.g `onOpen={ () => { this.changeAccordion( 'closed' ) }`

Reply:
good pick up 😅